src/wizard/WizardStep1.tsx

```diff
@@ -10,68 +10,103 @@ import {
   Grid,
   Text,
 } from '@patternfly/react-core';
+import { PushApplication } from '@aerogear/unifiedpush-admin-client';
 
 import { ActionBox } from './ActionBox';
+import { ApplicationListConsumer } from '../context/Context';
+import { CreateApplicationWizard } from '../application/wizard/CreateApplicationWizard';
+
+interface State {
+  openCreateAppWizard: boolean;
+}
+
+export class WizardStep1 extends Component<{}, State> {
+  constructor() {
+    super({});
+    this.state = {
+      openCreateAppWizard: false,
+    };
+  }
 
-export class WizardStep1 extends Component<{}> {
   render = (): React.ReactElement => {
```

Comment:
You imported `ReactElement` so you can change this
```suggestion
  render = (): ReactElement => {
```